.prow.yaml

```diff
@@ -319,6 +319,28 @@ postsubmits:
           requests:
             cpu: 2
             memory: 2Gi
+      - name: nephio-image-signer
+        image: nephio/releaser:latest
```

Comment:
instead of using the `latest` tag, it would be better to use a fixed tag? What do you all think?

Reply:
I second that, those need to be versioned

Reply:
In our releaser image build process, we're currently using the latest tag along with a tag that includes the build ID.
https://github.com/nephio-project/test-infra/blob/70edb080cf5ab26051f9aeefe503e723f3d12fc5/.prow.yaml#L457-L458

So, which tag `${BUILD_ID}` or `${PULL_BASE_REF}` is the recommended for the fix/versioned purpose?

Reply:
@radoslawc what is your suggestion?

Reply:
@anurag-rajawat @vjayaramrh BUILD_ID is more universal as it points to Prow job and logs from build can be viewed, PULL_BASE_REF gives you tag (or branch) and is used for release builds.